### PR TITLE
[Add catalogue and static information]

### PR DESCRIPTION
**Pull request by kboskin**
- Add apps to the apps list
- Move dependencies to a requirements file
- Add catalog entities
- Add static information entities

### CODE REVIEW

File: catalogue/models.py [hunk 1]
```diff
@@ -1,3 +1,18 @@
+from django.contrib.postgres.fields import ArrayField
 from django.db import models
 
+
 # Create your models here.
+class Category(models.Model):
+    category_name = models.CharField(max_length=30)
+    category_description = models.CharField(max_length=30)
+    parent_category_id = models.ForeignKey("self", on_delete=models.CASCADE)
```
Comment (oshevelo) [inline catalogue/models.py]:
```suggestion
    parent_category = models.ForeignKey("self", on_delete=models.CASCADE)
```
FK is OBJECT not integer field

File: catalogue/models.py [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+
+
+class StaticItem(models.Model):
+    category_id = models.ForeignKey(Category, on_delete=models.CASCADE)
```
Comment (oshevelo) [inline catalogue/models.py]: same shit as parent_category_id

File: catalogue/models.py [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+    html_content = models.TextField()
+
+
+class Catalogue(models.Model):
+    categories = ArrayField(models.IntegerField(blank=True), blank=True)
```
Comment (oshevelo) [inline catalogue/models.py]: many-to-many why array?

File: static_information/models.py [hunk 4]
```diff
@@ -1,3 +1,19 @@
 from django.db import models
 
+
 # Create your models here.
+
+class InformationCategory(models.Model):
+    information_cat_id = models.BigAutoField(primary_key=True)
```
Comment (oshevelo) [inline static_information/models.py]: same shit as parent_category_id

File: static_information/models.py [hunk 5]
```diff
@@ -1,3 +1,19 @@
...
+    category_name = models.CharField(max_length=30)
+
+
+class InformationItem(models.Model):
+    information_cat_id = models.ForeignKey(InformationCategory, on_delete=models.CASCADE)
```
Comment (oshevelo) [inline static_information/models.py]: same shit as parent_category_id

File: static_information/models.py [hunk 6]
```diff
@@ -1,3 +1,19 @@
...
+    content_path = models.CharField(max_length=255)
+
+
+class InlineInformationItem(models.Model):
+    information_cat_id = models.ForeignKey(InformationCategory, on_delete=models.CASCADE)
```
Comment (oshevelo) [inline static_information/models.py]: same shit as parent_category_id

File: static_information/models.py [hunk 7]
```diff
@@ -1,3 +1,19 @@
...
+    information_cat_id = models.BigAutoField(primary_key=True)
+    category_name = models.CharField(max_length=30)
+
+
+class InformationItem(models.Model):
+    information_cat_id = models.ForeignKey(InformationCategory, on_delete=models.CASCADE)
+    content_path = models.CharField(max_length=255)
+
+
+class InlineInformationItem(models.Model):
```
Comment (oshevelo) [inline static_information/models.py:20]: what do you need this class for?

Comment (kboskin) [reply static_information/models.py:20]: This class to reflect the "static" widgets, which might be a case in static section

Screenshot 2022-03-23 at 16 29 55

**Pull request merged**